src/components/ConsoleComponent.js

```diff
@@ -24,9 +46,33 @@ const theme = createMuiTheme({
 export default function Console(props) {
   const classes = useStyles();
 
+  const toggleExpandConsole = () => {
+    props.setExpandConsole(!props.expandConsole);
+  };
+
   return (
     <ThemeProvider theme={theme}>
-      <Box className={classes.box} overflow="scroll">
+      <Box
+        className={classes.consoleControls}
+        style={{ borderBottom: !props.expandConsole ? '6px solid #2c4f85' : '' }}
```

Comment:
It looks like the border that's on the collapsed console is 4px, as specified by the `box` class above. Is there a reason for changing between the two sizes? If the border is causing headaches, I think we could remove it if we wanted.

Reply:
I was just going for a height balance on the screen to prevent scroll! 

I definitely think that the border is easily salvageable if we really want that in there, but I'm at a loss as to how to we'd prevent the scroll on the entirety of the page without making these components resize in a weird way like they're doing now. I'd love to brainstorm with you on this separately to see if we can come up with a workaround!

Reply:
Yeah definitely! I had one idea about a slightly different approach to fixing the heights, but I tried once quickly yesterday and wasn't successful. I can give it one more shot or work with you to try for a bit!